packages/fx-core/src/plugins/resource/aad/index.ts

```diff
@@ -35,7 +35,12 @@ export class AadAppForTeamsPlugin implements Plugin {
     if (solutionSettings?.migrateFromV1) {
       return false;
     }
-    return solutionSettings.hostType === HostTypeOptionAzure.id;
+    return (
+      solutionSettings.hostType === HostTypeOptionAzure.id &&
+      // For scaffold, activeResourecPlugins is undefined
```

Comment:
Have we confirmed this is a contract between resource plugin and solution plugin / fx-core?

Reply:
Yeah, I have confirmed with @jayzhang when updating Simple Auth plugin.